ios/RNTempiFFT/TempiAudioInput.swift

```diff
@@ -248,3 +248,8 @@ private func DCRejectionFilterProcessInPlace(_ audioData: inout [Float], count:
     }
 }
 
+
+// Helper function inserted by Swift 4.2 migrator.
+fileprivate func convertFromAVAudioSessionCategoryArray(_ input: [AVAudioSession.Category]) -> [String] {
```

Comment:
Shouldn't need this! Stupid swift 4.2 migrator added this but it's really not necessary!